resources/test/index.html

```diff
@@ -0,0 +1,2 @@
+<!DOCTYPE html>
+<html><html><head><title>Desdemona</title><meta charset="utf-8"><meta content="width=device-width, initial-scale=1" name="viewport"><link href="/css/main.css" rel="stylesheet" type="text/css"></head><body><div id="app"></div><script src="/js/main.js" type="text/javascript"></script></body></html></html>
```

Comment:
Because it had been in a single line in the unit test that verified loading the index, and there was no way to break it up into multiple lines to pass eclint without breaking the test.